### PR TITLE
Add max limit pagination

### PR DESCRIPTION
**Pull request by itajaja**
Useful for setting a hard limit on the response page size

### CODE REVIEW

File: tests/test_pagination.py [hunk 1]
```diff
@@ -103,6 +110,23 @@ def data(db, models):
...
+            'id': '1',
+            'size': 1,
+        },
+        {
+            'id': '2',
+            'size': 2,
+        },
+    ])
+    assert get_meta(response) == {
+        'has_next_page': True
```
Comment (itajaja) [inline tests/test_pagination.py]: not sure if this is correct.

Comment (taion) [inline tests/test_pagination.py]: eh, seems fine

**Comment by itajaja** [comment]
tests are on 🔥 . some breaking change somewhere?

**Comment by taion** [comment]
`test_swagger.py` breaks everything after it

**Comment by itajaja** [comment]
uh, how do you know is `test_swagger`? There are some tests that run fine after it

**Comment by itajaja** [comment]
✂️  release?

**Pull request merged**